knitr/planetary_motion/planetary_motion3.rmd

```diff
@@ -0,0 +1,737 @@
+---
+title: |
+  | Bayesian Model of Planetary Motion: exploring ideas for a modeling workflow when dealing with ordinary differential equations and multimodality
```

Comment:
Was I supposed to review this file? (`planetary_motion3.rmd` as opposed to `planetary_motion.rmd`)

Reply:
nope. you reviewed the right file!